students/jinsup/westagram/users/views.py

```diff
@@ -28,12 +28,15 @@ def post(self, request):
             if User.objects.filter(email = email).exists():
                 return JsonResponse({'message' : 'Email_Already_Exists'}, status=400)
 
+            hashed_password = bcrypt.hashpw(data['password'].encode('utf-8'), bcrypt.gensalt())
+            hashed_password = hashed_password.decode('utf-8')
```

Comment:
변수로 만들어서 가독성을 높인점 잘 하셨습니다.
대신에 같은 변수를 사용 안 했으면 더 좋았을 것 같습니다.
```suggestion
            hashed_password = bcrypt.hashpw(data['password'].encode('utf-8'), bcrypt.gensalt())
            decoded_password = hashed_password.decode('utf-8')
```

Reply:
넵 수정하겠습니다~